src/core/infrastructure/entities/Message.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, Unique, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { UserRecord } from "./User";
+
+@Entity({ name: "tribe_messages" })
```

Comment:
Entity names need to match up with the files. You can't have a file and an entity called `Message` and call the table `tribe_messages`. This creates confusion